schema/dereferenced-release-schema.json

```diff
@@ -2173,6 +2173,29 @@
                       "string",
                       "null"
                     ]
+                  },
+                  "weight": {
+                    "title": "Weight",
+                    "description": "The weight of the item unit.",
+                    "type": "object",
+                    "properties": {
+                      "name": {
+                        "title": "Name",
+                        "description": "The name of the weight, using UN/CEFACT Common Codes for Units of Measurement.",
+                        "type": [
+                          "string",
+                          "null"
+                        ]
+                      },
+                      "value": {
```

Comment:
I think this should be either `amount` or `quantity`, since `value` fields in OCDS otherwise refer to the `Value` object.

Reply:
Hmm, you are right, following the same logic, shouldn't we change `unit` to another name too?
cc @jpmckinney

Reply:
I think it's okay to use `unit` again, because there is no better term (unless anyone has a better idea).

A unit of weight (or length, etc.) is basic and standardized, such that it can be expressed as a single code string. (We could of course add fields to reference the standards body, fields from [QUDT](http://www.qudt.org/), etc., but such details can be looked up in other datasets based on the code only.)

A unit of goods is neither basic nor standardized, requiring multiple data elements, so it is expressed as an object.

It is the same term `unit`, but in different contexts, so I think it's fine for the value to differ. We *could* disambiguate by adding prefixes like `weightUnit`, but that goes against our [style guide](https://ocds-standard-development-handbook.readthedocs.io/en/latest/meta/schema_style_guide.html#field-and-code-names).

As for `value`, I agree that `quantity` is better, which is commonly used with units of measure. `value` has a tendency toward overuse anyway, so if we can reduce confusion, that is better.

Reply:
Or, we could use `unitID`, since it's identifying the unit, for lookup in other datasets.

Reply:
Actually, we probably want `unit` to be an object with `scheme` and `id`, so that people can choose between UN/CEFACT, QUDT, [IEC 61360](https://en.wikipedia.org/wiki/IEC_61360), [UCUM](https://ucum.org/ucum.html), etc.